betterproto/plugin.py

```diff
@@ -373,8 +216,207 @@ def generate_code(request, response):
         init = response.file.add()
         init.name = str(init_file)
 
-    for filename in sorted(output_paths.union(init_files)):
-        print(f"Writing {filename}", file=sys.stderr)
+    for output_package_name in sorted(output_paths.union(init_files)):
```

Comment:
this diff sucks

Reply:
yeah, it went pretty bad after splitting up code into methods and renaming a bunch of vars :-/

should happen less and less as we get the code more organized